frontend/src/container/SideNav/NavItem/NavItem.tsx

```diff
@@ -24,7 +24,7 @@ export default function NavItem({
 			onClick={(event): void => onClick(event)}
 		>
 			<div className="nav-item-active-marker" />
-			<div className="nav-item-data">
+			<div className={cx('nav-item-data', isBeta ? 'beta-tag' : '')}>
```

Comment:
@SagarRajput-7 can you please use the same code as we used in other place:  https://github.com/SigNoz/signoz/pull/5731/files
frontend/src/container/SideNav/NavItem/NavItem.tsx